github/github.go

```diff
@@ -1192,12 +1267,35 @@ const (
 )
 
 // category returns the rate limit category of the endpoint, determined by Request.URL.Path.
-func category(path string) rateLimitCategory {
+func category(method string, path string) rateLimitCategory {
```

Comment:
Please update comment on line 1269: "determined by HTTP method and Request.URL.Path."

Also, please remove the extra "string":

```go
func category(method, path string) rateLimitCategory {
```

Reply:
It looks like you still need to remove the extra "string". Please see my last comment above.